### ISSUE TITLE
SVG Logo

### ISSUE THREAD

**Issue by sjehuda** [miranda-ng/miranda-ng#3074]
SVG logo for this project. 
![miranda-ng](https://user-images.githubusercontent.com/19249089/169428955-34f33c73-fad5-4125-9d73-5e09813ec599.svg)
I hope you like it.

**Comment by georgehazan** [comment]
@sjehuda hi, yes, it's adorable.
if you could use the shades of cyan, like in our current PNG logo, we'd replace the logo at the site & forum with your picture

**Comment by sjehuda** [comment]
Shalom dear George, long time no speak (:

1) This logo is based on the Miranda IM logo.
2) Edits were made in order to include Miranda NG in an XMPP related project.
3) I'm an attorney, not a graphics desiner, so it's tough on me to get this right.

Just kidding, I'll look into it and see what can be done.
Shavuah Tov, George!

On May 20, 2022 1:44:18 PM UTC, George Hazan ***@***.***> wrote:
***@***.*** hi, yes, it's adorable.
>if you could use the shades of cyan, like in our current PNG logo, we'd replace the logo at the site & forum with your picture
>

**Comment by sjehuda** [comment]
Can someone please take over this task.
I'm not skillful enough with Inkscape to make something reasonable in a short time.

**Comment by sjehuda** [comment]
Is there a higher resolution of this logo? https://miranda-ng.org/logo.png

**Comment by georgehazan** [comment]
I don't think so, there're some variants of this picture without blue background.

**Comment by georgehazan** [comment]
![apple-touch-icon](https://github.com/user-attachments/assets/1a04a3d9-18b6-4e9c-8811-15d673fd7ba8)
This one, for example

**Comment by sjehuda** [comment]
Miranda NG has been added to FASI as a featured XMPP client.

https://git.xmpp-it.net/sch/FASI/src/branch/main/img/miranda.svg

https://git.xmpp-it.net/sch/FASI/src/commit/8675f295e62bfb49377137408d5fb4f901d3a340/clients.toml#L409
